lib/memoizable/memory.rb

```diff
@@ -75,7 +75,9 @@ def fetch(name)
     #
     # @api public
     def key?(name)
-      @memory.key?(name)
+      @monitor.synchronize do 
+        @memory.key?(name)
+      end
```

Comment:
__comment:__ It's weird that mutant didn't have a mutation that forced a test to be written here.